ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -938,6 +959,13 @@ static PrjFS_Result HandleFileNotification(
         isDirectory,
         notificationType,
         nullptr /* destinationRelativePath */);
+    
+    if (partialFile && PrjFS_NotificationType_FileModified == notificationType)
```

Comment:
"&& result == PrjFS_Result_Success"?

Reply:
As discussed offline, I'll see if we can\should make `HandleFileNotification` have a `void` return type.

Reply:
@sanoursa I checked and the `OnNotifyOperation` operation is also used for `PreDelete` (which might have a non-success result).

I think we could change this behavior, but the scope is outside of this PR.  For now I'll add a comment, let me know if you'd like me to file a follow up issue.